FormControlRange/explainer.md

```diff
@@ -426,15 +434,11 @@ const textarea = document.querySelector("#messageArea");
 const wordRange = new FormControlRange();
 wordRange.setFormControlRange(textarea, 0, 5);
 
-// Apply highlight to that live range.
+// Bind a highlight to that live range once.
```

Comment:
nit: what does "once" mean to emphasize here? Seems clearer to just say "Bind a highlight to that live range".